pennylane/transforms/decompose.py

```diff
@@ -580,6 +628,26 @@ def circuit():
             >>> new_tape.operations
             [RX(0.25, wires=[1]), CZ(wires=[0, 1]), RX(-0.25, wires=[1]), CZ(wires=[0, 1])]
 
+        **Weighted Gate SSets**
+
+        With the graph based decomposition enabled, gate weights can be provided in the ``gate_set`` parameter. i.e.
+
+        .. code-block:: python
+
+            @partial(qml.transforms.decompose, gate_set={qml.Toffoli: 1.23, qml.RX: 4.56, qml.RZ: 0.01, qml.H: 420})
```

Comment:
```suggestion
            @partial(
                qml.transforms.decompose, 
                gate_set={qml.Toffoli: 1.23, qml.RX: 4.56, qml.RZ: 0.01, qml.H: 420}
            )
```

just to get rid of the horizontal scroll bar